### ISSUE TITLE
Tests failing due to missing json files

### ISSUE THREAD

**Issue by jccaicedo** [broadinstitute/DeepProfiler#60]
This is the error:

```
target = <deepprofiler.dataset.target.MetadataColumnTarget object at 0x7fa5f7114ef0>, values = [5, 81, 83, 17, 71, 12, ...]

    def test_init(target, values):
        field_name = 'test'
        shuffle(values)
        assert target.field_name == field_name
>       assert len(target.index) == len(values)
E       assert 9 == 10
E        +  where 9 = len({5: 1, 8: 2, 12: 3, 17: 4, ...})
E        +    where {5: 1, 8: 2, 12: 3, 17: 4, ...} = <deepprofiler.dataset.target.MetadataColumnTarget object at 0x7fa5f7114ef0>.index
E        +  and   10 = len([5, 81, 83, 17, 71, 12, ...])

tests/dataset/test_target.py:24: AssertionError
```

**Comment by santient** [comment]
I fixed this in branch issues/36 - I missed it when committing this branch. Merging 36 and 37 should fix it

**Comment by jccaicedo** [comment]
OK. Great! Yes, I confirm it's fixed after merging 36 and 37.
Now there is another test that fails:

```
    def test_read_dataset():
>       with open('deepprofiler/examples/config/learning.json', 'r') as f:
E       FileNotFoundError: [Errno 2] No such file or directory: 'deepprofiler/examples/config/learning.json'
```
The JSON file was not added to the commit. Can you add it as a fix for this thread?

**Comment by santient** [comment]
Will do. Though I have to edit it to work with relative paths

**Comment by jccaicedo** [comment]
Sounds good! Adding @massachusett to this thread cause there are a couple tests that fail because of a missing JSON file and absolute paths:

```
def test_split_index():                                                                                                                                      [0/1392]
>       open_file = open("/Users/msmith/DeepProfiler/deepprofiler/examples/config/data.json")
E       FileNotFoundError: [Errno 2] No such file or directory: '/Users/msmith/DeepProfiler/deepprofiler/examples/config/data.json'
```
And

```
>           mkdir(name, mode)
E           PermissionError: [Errno 13] Permission denied: '/Users'
```

Let's discuss potential fixes for this relative paths issue.

**Comment by santient** [comment]
Fixed with 5b622e8e42e234554da18e36560bf120cd18a409
